tests/corelib_2/symbol_map_helper.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dart.test.symbol_map_helper;
+
+// TODO(ahe): Update map literals to avoid this method.
+Map<Symbol, dynamic> symbolMapToStringMap(Map<String, dynamic> map) {
```

Comment:
🤷‍♂️ I tried to do the merge mostly mechanically. I think there are other yet-to-be-migrated tests that use this library, so it might make it harder to migrate those if I rename this now.